fix: format and wording adjustments in workspaces, contributors and repo insights sections (#262)
open-sauced/docs: docs/contributors/contributors-guide.md

- **Contributions**: This tab includes a graph showing the number of PRs you have opened, [the velocity of your PRs](../welcome/glossary.md#pr-velocity), and the number of repositories you have contributed to. Underneath the graph is a "Latest PRs" chart, which shows a list of current and recent PRs. This tab is helpful if you want to showcase a steady and consistent open source contribution experience.
- **Connections**: For pro accounts, you can reach out to other contributors and maintainers to collaborate on projects or to learn from them.
- **Recommendations**: This tab includes a list of open source projects that are chosen based on the programming languages you picked in the "Interests" section of your public profile. It is useful if you're looking for new projects to contribute to.

## Explore: Finding Your Next Open Source Project

The "Explore" tab is a great place to start if you're looking for a new open source project to contribute to or want to find repositories for potential job opportunities. Here, you can find a list of open source projects and filter them by language, top repos, recent, most active, and more.

> Once you've selected your filters, you'll be able to see the contributors, spam, and activity levels of the repository. Understanding a repository's activity level and PR velocity can help you determine if it's a good fit for you to contribute to and what to expect when you submit a PR.

Author: Just a note:

We might want to revisit this. I'm not sure here are we talking about the higher view (Contributors, Spam, Merged PRs, and Unlabeled PRs) or the data in the Repositories.

![contributors insight list](../../static/img/contributors-insights.png)

:::tip

Want to learn more about your peers? You can select and add them to a Contributor Insight Page.

:::

To learn and understand more about the data provided, see [Understanding Repository Insights Data](../maintainers/understanding-repo-insights.md).

#### Activity

The "Activity" tab shows a graph of the last time each contributor created their PR and how many lines of code they've touched.

It also provides more detailed information on each contributor when you hover over their image, including their latest PRs and repositories they've contributed to. Clicking their image will bring you to their profile on OpenSauced. This information can help you find potential repositories to contribute to and peers to collaborate with or ask for mentorship.

> ![activity graph](../../static/img/activity-repos-insights.png)

Author: I added the Activity tab subsection, as well as an image fo the tab.
I'm not sure if you're intentionally leave it out. So please let me know if I need to remove it.

I also added the same subsection for the Maintainers Guide. Wordings are the same. The difference is the purpose of this tab for contributor and maintainer, which I need you to check.

Thanks! 😊

- **Learn about what other contributors are doing in open source**: You can learn where other contributors are contributing, what they're working on, and how active they are in the open source community.
- **See who's using the OpenSauced Highlights feature**: You can see who's using the Highlights feature and show them support by giving an emoji reaction.
- **Track contributors for job opportunities**: If you're looking for a job, you can track the activity of the contributors you're interested in working with. This helps you understand the activity level of the repositories and the types of contributions being made, and you can even identify which company you want to contribute to.

### Creating a New Contributor Insights Page

1. Click the "+" next to "Insights" in your sidebar.
2. Select "New Contributor Insight" to start a new Contributor Insight Page.
3. Give your page a name.
4. Add contributors to your page by searching for their GitHub username, syncing your GitHub Team, or importing your GitHub Following.

> 5. Choose your page visibility. You can choose to make your list public or private. By default, it is set to public. To create a private page, you can upgrade your account to a paid plan.

Reviewer: Sure. I just wrote some issues to get that changed. Nice catch.

> 5. Choose your page visibility. You can choose to make your list public or private. By default, it is set to public. To create a private page, you can upgrade your account to a paid plan.

Author: If we'll change it as per your issue on app, we can leave it as is. Otherwise, you can just commit them if it's a go. 😊

![contributors insight list](../../static/img/contributors-insights.png)

:::tip

Want to learn more about your peers? You can select them and add them to a list.

:::

> To learn more about these features, see [Understanding Repository Insights Data](../maintainers/understanding-repo-insights.md).

Author: As we're talking about how to read and understand the data, we might be able to clarifiy this with wording adjustment as below:

```suggestion
To learn and understand more about the data provided, see [Understanding Repository Insights Data](../maintainers/understanding-repo-insights.md).
```

Any other idea on the wording?

If we agreed on the wording, we might as well will reword the same sentence in the maintainer guide.

Reviewer 2: How about this way @adiati98? It’ll clarify to current and potential users to expand their knowledge a bit more.

```suggestion
Want to learn other ways to use these features? Check out [Understanding Repository Insights Data](../maintainers/understanding-repo-insights.md).
```

Author: In my understanding, what we want to say here is for folks to learn about how to read and understand the data provided in each tab. And these are not part of features.

Would love to hear @BekahHW opnion about this.

Reviewer: @adiati98's correct here.

- **Track personal contributions**: If you create a list of your personal contributions, you can track them and see how you're doing over time, as well as get a breakdown of the type of contributions you're making, including commits, created PRs, reviewed PRs, and created issues.
- **Learn about what other contributors are doing in open source**: You can learn where other contributors are contributing, what they're working on, and how active they are in the open source community.

> - **See who's using the OpenSauced Highlights feature**: You can see who's using the Highlights feature and show them support by giving an emoji reaction.

Reviewer 2: [improvement] Here, it would be beneficial to point out a aspect of open source & how the OpenSauced feature can facilitate it. What do you think @adiati98?
```suggestion
- **Support other contributors**: Networking with others is key to thriving in open source. You can do this by posting a emoji  on someone’s highlight in the Highlights section.
```

Reviewer: @CBID2 that's out of the scope of this PR.
